core/src/main/java/org/frankframework/jdbc/MessageStoreSender.java

```diff
@@ -84,6 +85,9 @@ public void configure() throws ConfigurationException {
 			paramList.configure();
 		}
 		setType(StorageType.MESSAGESTORAGE.getCode());
+		if (StringUtils.isBlank(getSlotId())) {
+			throw new ConfigurationException("'slotId' has to be configured");
```

Comment:
Ik gebruik altijd blokhaken, het is niet echt een standaard maar volgens mij doen we dat wel overal.
```suggestion
			throw new ConfigurationException("[slotId] has to be configured");
```

En mocht je de XSD willen verbeteren kan je de methode overschrijven met de `@Mandatory` annotatie. Dan is hij ook in de XSD verplicht.